op-acceptance-tests/tests/interop/loadtest/interop_load_test.go

```diff
@@ -27,8 +30,27 @@ func TestExecFromSameAddressInALoop(gt *testing.T) {
 	}
 	t := devtest.SerialT(gt)
 	sys := presets.NewSimpleInterop(t)
-	eoaA := sys.FunderA.NewFundedEOA(eth.MillionEther)
-	eoaB := sys.FunderB.NewFundedEOA(eth.MillionEther)
+
+	// Use proxyd if available.
+	l2ELA := sys.L2ELA
+	l2ELB := sys.L2ELB
+	if proxydsA := match.Proxyd.Match(sys.L2ChainA.Escape().L2ELNodes()); len(proxydsA) > 0 {
+		l2ELA = dsl.NewL2ELNode(proxydsA[0])
+	}
+	if proxydsB := match.Proxyd.Match(sys.L2ChainB.Escape().L2ELNodes()); len(proxydsB) > 0 {
+		l2ELB = dsl.NewL2ELNode(proxydsB[0])
+	}
+
+	// Setup EOAs. We are careful to use the specific ELs configured above to ensure we
+	// hit proxyd (if configured) to simulate real load.
+	skA, err := crypto.GenerateKey()
+	t.Require().NoError(err)
+	skB, err := crypto.GenerateKey()
+	t.Require().NoError(err)
+	eoaA := dsl.NewEOA(dsl.NewKey(t, skA), l2ELA)
+	eoaB := dsl.NewEOA(dsl.NewKey(t, skB), l2ELB)
+	sys.FaucetA.Fund(eoaA.Address(), eth.MillionEther)
+	sys.FaucetB.Fund(eoaB.Address(), eth.MillionEther)
```

Comment:
If we're planning to run this test on sysext networks we might want to defer a function to send remaining eth back after the test. Note that it can be a bit tricky to calculate the value such that there's always enough gas for the transaction.

Reply:
Interesting. Perhaps this should be the system's or orchestrator's responsibility to ensure it happens for all tests?

Reply:
Or maybe the faucet can include a `t.Cleanup` in `Fund`.

Reply:
https://github.com/ethereum-optimism/optimism/issues/15930